config/initializers/job_configurations.rb

```diff
@@ -217,6 +217,12 @@
         class: 'ThreatMetrixJsVerificationJob',
         cron: cron_1h,
       },
+      # Batch up IRS Attempts API events
+      irs_attempt_events_aggregator: {
+        class: 'IrsAttemptsEventsBatchJob',
+        cron: cron_1h,
+        args: -> { [Time.zone.now - 1.hour] },
+      },
```

Comment:
this will get run in all environments regardless of if attempts api is enabled or not, which seems not good. let's either:
1. add a check inside the job to bail if the api is not enabled
2. only add the job to the schedule if the api enabled

and I know we have a lot of configs, but it seems like we may want to have a separate feature flag for just the attempts API background job so that we can smoothly transition to it if/when we are ready

Reply:
I think the first line inside the job checks if IRS attempts API is enabled and returns nil if it isn't, so it should only continue if it is enabled. 

With that said, I do like the idea of ALSO not scheduling the job unless it is enabled, since I don't think that makes much sense. Plus, even though its probably not a huge resource tax, it's one less job the scheduler has to deal with if it doesn't need to.